### PR TITLE
Readme - updated docu link

### PR DESCRIPTION
**Pull request by arbadacarbaYK**
changed docs.lnbits.org - links to new wiki

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Wallets can be easily generated and given out to people at events. "Go to this
 If you like this project [send some tip love](https://legend.lnbits.com/paywall/GAqKguK5S8f6w5VNjS9DfK)!
 
 
-[docs]: https://docs.lnbits.org/
+[docs]: [https://docs.lnbits.org/](https://github.com/lnbits/lnbits/wiki)
```
Comment (callebtc) [inline README.md:72]:
```suggestion
[docs]: https://github.com/lnbits/lnbits/wiki
```

Comment (callebtc) [reply README.md:72]: I think this is what you meant?

**Comment by callebtc** [comment]
Hey @arbadacarbaYK :) I changed one line because that's what I thought you meant. Could you check if this is ok with you?

File: README.md (hunk 1 above)
Comment (arbadacarbaYK) [reply README.md]: yep, thanks

**Comment by prusnak** [comment]
remote links to contain `https://` protocol, otherwise markdown thinks you are linking to a local file. fix in https://github.com/lnbits/lnbits/pull/1572

**Pull request merged**